### PR TITLE
Updated environments; updated sample data to fit loader defaults; added additional sample data

### PR DESCRIPTION
**Pull request by AlexandreHutton**
Fixes conda environment.yml and requirements.txt. Fixed download_data.py + sample data to match one another + fit expected structure in RAMP (sample/train + sample/test).

### CODE REVIEW

File: .github/workflows/testing.yml [hunk 1]
```diff
@@ -3,9 +3,11 @@ on:
   push:
     branches:
       - master
+      - main
   pull_request:
     branches:
       - master
+      - main
```
Comment (agramfort) [inline .github/workflows/testing.yml:10]:
```suggestion
      - main
  pull_request:
    branches:
      - main
```

File: config.py [hunk 2]
```diff
@@ -38,6 +39,12 @@
 
 
 try:
+    if(not os.path.exists(training['dir_name'])):
```
Comment (agramfort) [inline config.py:42]:
```suggestion
    if not os.path.exists(training['dir_name']):
```

File: config.py [hunk 3]
```diff
@@ -38,6 +39,12 @@
...
+        train_path = join(os.path.dirname(__file__), data_path, 'train')
+        if(os.path.exists(train_path)):
```
Comment (agramfort) [inline config.py:44]:
```suggestion
        if os.path.exists(train_path):
```

File: config.py [hunk 4]
```diff
@@ -58,5 +71,6 @@
                                   label_names=testing['label_names'],
                                   batch_size=testing['batch_size'])
 except BIDSValidationError:
+    print('Warning: BIDS default path is not valid. Consider modifying config.py to match your data structure.')
     # default testing path not valid; ignore
     pass
```
Comment (agramfort) [inline config.py:76]:
```suggestion
    pass

```

**Review by agramfort — COMMENTED** [review]
@AlexandreHutton can you see how to make CIs green?

note that you should not need to commit any nii.gz files as the data should be added to folder with the download_data.py script

**Comment by AlexandreHutton** [comment]
I'm working on it.

The .nii.gz are 1-voxel images (i.e. very small), and they are necessary for the tests that load BIDS data.
download_data.py won't download data in a usable format since we're bound by our data distribution policy. The data is encrypted and the password is distributed once users agree to usage terms.

The current build errors come from RAMP's 'import_module_from_source', which doesn't seem to allow for importing modules from uninstalled packages (i.e., the current repo). I need to give some thought about how to best restructure that.

I'll close the draft for now.

**Comment by agramfort** [comment]
>
> I'm working on it.
>
ok

> The .nii.gz are 1-voxel images (i.e. very small), and they are necessary
> for the tests that load BIDS data.
> download_data.py won't download data in a usable format since we're bound
> by our data distribution policy. The data is encrypted and the password is
> distributed once users agree to usage terms.
>
we could make the password a secret of the github action that runs the CI
so we can actually check it's working
on a random machine.

https://docs.github.com/en/actions/security-guides/encrypted-secrets

> The current build errors come from RAMP's 'import_module_from_source',
> which doesn't seem to allow for importing modules from uninstalled packages
> (i.e., the current repo). I need to give some thought about how to best
> restructure that.
>
yes. One option is to put everything in problem.py

**Pull request closed without merging**